content/docs/querying/functions.md

```diff
@@ -80,6 +86,55 @@ and value across all series in the input vector.
 `floor(v instant-vector)` rounds the sample values of all elements in `v` down
 to the nearest integer.
 
+## `histogram_quantile()`
+
+`histogram_quantile(φ float, b instant-vector)` calculates the φ-quantile (0 ≤ φ
```

Comment:
My concern is that most people are not good at math and are scared by anything that's more complex than addition and multiplication.

Using Greek letters is going to make many users discard this function (and possibly all of Prometheus) out of hand.

This documentation needs to not do that, we need to aim for the lowest-common denominator for at least the first paragraph.